episodes/03-feature-branch.md

```diff
@@ -53,6 +53,26 @@ $ git clone git@github.com:metoffice/git-training-demo.git
 
 Make sure to navigate to the `Desktop` folder first.
 
+::: spoiler
+
+### Commands that communicate between repositories
+
+In the Introduction to Version Control with Git and GitHub
+course you learnt that only certain git commands
+communicate between repositories.
+
+They were:
+
+- `git fetch`: To fetch changes from a remote.
+- `git pull`: To fetch changes from a remote
+  and merge them into a branch.
+- `git push`: To push changes to a remote.
```

Comment:
Technically there are things like `git ls-remote`, variants of `git submodule`, `git remote update`, etc. Maybe we should say "There are also other git commands that interact with remote repositories (e.g. `git ls-remote`), but none that appear in this course."